gc/base/standard/Scavenger.cpp

```diff
@@ -4403,10 +4632,14 @@ MM_Scavenger::scavengeComplete(MM_EnvironmentBase *envBase)
 bool
 MM_Scavenger::scavengeIncremental(MM_EnvironmentBase *env, int64_t scavengeIncrementEndTime)
 {
+	Assert_MM_mustHaveExclusiveVMAccess(env->getOmrVMThread());
 	bool result = false;
 	bool timeout = false;
 
 	while (!timeout) {
+//		OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
```

Comment:
I intend to remove them (already did in my work space). If I have to modify this PR again for some other reasons, I'll do it as part of it, if not, I'll defer it for next PR (to avoid nowdays rather slow Travis compile)